rocket_sqlite_oracle/Rocket.toml

```diff
@@ -0,0 +1,2 @@
+[default.databases.example_signatures]
```

Comment:
I think it is quite clear that this is an example, no need to repeat ourselves in having "example" in all kinds of names.